common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -1091,6 +1091,12 @@
     ],
     "sqlState" : "KD010"
   },
+  "DATA_SOURCE_UNSUPPORTED_DML_OPERATION" : {
```

Comment:
Maybe subclasses for each operation is better?
Operations are something like
'CREATE TABLE', 'ALTER NAMESPACE', etc

Reply:
It is common to name the operation: "ALTER TABLE', "UPDATE" as a parameter.